lms/templates/instructor/instructor_dashboard_2/instructor_dashboard_2.html

```diff
@@ -143,6 +144,18 @@ <h3 class="hd hd-3" id="header-${ section_data['section_key'] }">${ section_data
   </div>
 </div>
 
+% if configuration_helpers.get_value('CUSTOM_THEME_DISPLAY_SENSITIVE_DATA_MSG_FOR_DASHBOARD', settings.FEATURES.get('CUSTOM_THEME_DISPLAY_SENSITIVE_DATA_MSG_FOR_DASHBOARD', True)):
+
+<script>
+$('[data-section="data_download"]').click(function(){
+    $( 'a[rel*=leanModal]' ).leanModal();
+    $( "#sensitive_data" ).click()
+    $( "#sensitive" ).attr("aria-hidden", "false");
+});
+</script>
+
+% endif
+
```

Comment:
Was necessary place this code here due to JS in `data_download.html` executes in all tabs, or I'm missing something of how can I execute JS in a single tab once the page is loaded?